docker/BUILD.bazel

```diff
@@ -77,7 +77,7 @@ genrule(
 
```

Comment:
You need to add `export_files(["PACE"])` to the top of `BUILD.bazel` right below `load("//bazel:common_settings.bzl", "string_flag")` so the `//docker` package has access to `PACE`, which exists outside of the `//docker` package. You can then test by running `bazel build //docker:push_pace_x86_64`.

Reply:
I'll fix this thanks Peter.